lib/rihanna/migration/upgrade.ex

```diff
@@ -116,6 +127,24 @@ defmodule Rihanna.Migration.Upgrade do
       """,
       """
       CREATE INDEX IF NOT EXISTS rihanna_jobs_enqueued_at_id ON rihanna_jobs (enqueued_at ASC, id ASC);
+      """,
+      """
+      DO $$
+          BEGIN
+              ALTER TABLE #{table_name} ADD COLUMN priority integer NOT NULL DEFAULT 50;
+          EXCEPTION
+              WHEN duplicate_column THEN
+              RAISE NOTICE 'column already exists in #{table_name}.';
+          END;
+      $$
+      """,
+      """
+      DO $$
+          BEGIN
+              DROP INDEX IF EXISTS rihanna_jobs_enqueued_at_id;
```

Comment:
@lpil Is this the drop index you were looking for?

Reply:
Also, looks like the `upgrade` doesn't always use the `#{table_name}`. I'll fix that to not hard code `rihanna_jobs`.

Reply:
oh yes, thanks! :)